### PR TITLE
Utility for extrating test summaries from test results.

### PR DESCRIPTION
**Pull request by gmarek**
cc @mikedanese @davidopp @wojtek-t @fgrzadkowski 

Assigning @mwielgus, as it'll be probably most convenient.

### CODE REVIEW

File: compare/src/scraper.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	"bytes"
+	"encoding/json"
+	"strings"
+
+	"k8s.io/kubernetes/test/e2e"
+
+	"github.com/golang/glog"
+)
+
+func ProcessSingleTest(scanner *bufio.Scanner, buildNumber int) (*e2e.LogsSizeDataSummary, *e2e.ResourceUsageSummary, *e2e.MetricsForE2E) {
```
Comment (mwielgus) [inline compare/src/scraper.go]: What is the input format?

Comment (gmarek) [inline compare/src/scraper.go]: Test output with embedded JSON data. See recent scalability test results.

**Comment by mwielgus** [comment]
LGTM

**Comment by gmarek** [comment]
Small lint/gofmt changes.

**Pull request merged**